packages/firestore/test/lite/integration.test.ts

```diff
@@ -2116,6 +2117,45 @@ describe('countQuery()', () => {
     });
   });
 
+  it('count with filter and order by', () => {
+    return withTestCollectionAndInitialData(testDocs, async collection => {
+      const query1 = query(
```

Comment:
nit: rename `query1` to `query_`. When the suffix is a number it implies that there is a `query2`, and maybe a `query3` later on; however, in this case, that is not true, which leads to confusion. By simply naming it `query_` the reader can easily, and correctly, assume that it is the only query in scope.